python/zygote.py

```diff
@@ -72,6 +74,7 @@
 # debug the problem.
 def try_dumps(obj, sort_keys=False, allow_nan=False):
     try:
+        zu.all_integers_within_limits(obj)
```

Comment:
Micro-nit: maybe a slightly-more descriptive name like `assert_all_integers_within_limits`?

Reply:
Done in https://github.com/PrairieLearn/PrairieLearn/pull/6980/commits/1d2f92670d5dc5699c47a66f85c111f7f5cae091